docs/src/code/algo/mofa.rst

```diff
@@ -0,0 +1,5 @@
+MOFA Algorithm
+===================
+
+.. automodule:: orion.algo.mofa.mofa
+   :members:
```

Comment:
```suggestion
   :members:

```
To fix doc8 issue.

Reply:
Addressed in https://github.com/Epistimio/orion/pull/864/commits/710af4da5865c02381c797fda7cd3b5e24b1c76d